apps/shipments/filters.py

```diff
@@ -146,9 +146,14 @@ class Meta:
 
 
 class TelemetryFilter(filters.filterset.FilterSet):
+    timestamp_gte = filters.IsoDateTimeFilter(field_name="timestamp", lookup_expr='gte')
+    timestamp_lte = filters.IsoDateTimeFilter(field_name="timestamp", lookup_expr='lte', )
```

Comment:
In this case, I would prefer `after` and `before`.  In my opinion those are easier to remember and are more immediately obvious what they do.

Reply:
by this did you mean just `after` and `before` or still maintain the relation to `timestamp_`?

Reply:
`after` and `before`